### ISSUE TITLE
Beautify Prompt Page

### ISSUE THREAD

**Issue by Caleb-Highsmith** [alejmedinajr/AI-Detector#41]
I think we have enough of a product here to start cleaning up the aesthetic of the prompt page (the main react page that pops up on startup). This will be a good time to try out the ChakraUI package for React.

**Comment by Caleb-Highsmith** [comment]
Current update, I got the new chakraui library installed and integrated into our project. Also added light and dark mode feature. Currently reorganizing page and adding icons for file upload buttons etc.

**Comment by Caleb-Highsmith** [comment]
Restructured multiple components. Tried adding a transition UI feature when the light dark mode is toggled but could not figure it out yet. Overall pretty happy with what the current state is but, want to spend a bit more time on this. Also added new model buttons that don't do anything but they will be able to specify which AI model they want to test against. Per @alejmedinajr request, I added the uploads folder to the git ignore. Committing and taking a break for now.
